src/release/breaking-changes/android-14-nonlinear-text-scaling-migration.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Android 14 nonlinear font scaling enabled after v3.14
+description: >
+  New Android 14 nonlinear font scaling feature is enabled in Flutter after v3.14.
+---
+
+## Summary
+
+Android 14 introduced nonlinear font scaling to 200%. It may change how your app 
+looks when the user changes the accessiblity text scaling in system preferences.
```

Comment:
**chore:** Use [semantic breaks](https://sembr.org/). Fix misspelling.
```suggestion
Android 14 introduced nonlinear font scaling up to 200%.
It may change how your app looks when the user changes
the accessibility text scaling in system preferences.
```